psydac/linalg/basic.py

```diff
@@ -1016,3 +1018,8 @@ def solve(self, rhs, out=None):
     @property
     def T(self):
         return self.transpose()
+
+    
+def is_real(x):
+    from numbers import Number
```

Comment:
Maybe it would be more appropriated to add this function to utilities.utils.py and then import it here?

Reply:
Good point. Originally this function was defined in `psydac.linalg.solvers` and only used in that module.

Reply:
Done in 4b0f821.